tbdex/validator/validator_test.go

```diff
@@ -0,0 +1,6 @@
+package validator_test
+
+// func TestValidate_Invalid(t *testing.T) {
```

Comment:
delete?

Reply:
Yee

Reply:
oops. i dont even remember commenting this out. uncommented: https://github.com/TBD54566975/tbdex-go/commit/8622cccbcbf1cac735c111df1dea32e5483bdd58